.travis.yml

```diff
@@ -1,7 +1,13 @@
 language: node_js
 node_js:
-- 5
+- 10
+<<<<<<< HEAD
```

Comment:
Looks like some merge conflicts are not resolved here.

Reply:
Whoops, that's odd. I'll get on that.